includes/Admin/Product_Categories.php

```diff
@@ -55,6 +55,19 @@ public function enqueue_assets() {
 	 */
 	public function render_add_google_product_category_field() {
 
+		$category_field = new Google_Product_Category_Field();
```

Comment:
As already mentioned in #1548, this feels weird. Should this class be abstract and contain static methods instead?

Reply:
I posted an answer in https://github.com/facebookincubator/facebook-for-woocommerce/pull/1548#discussion_r489114763.